doc/datasets/index.rst

```diff
@@ -254,6 +254,36 @@ features::
 
  _`Faster API-compatible implementation`: https://github.com/mblondel/svmlight-loader
 
+.. _external_datasets:
+
+Loading from external datasets
+==============================
+
+scikit-learn works on any numeric data stored as a numpy array or a pandas DataFrame, 
```

Comment:
Should we say that it supports numpy, pandas DataFrame, and scipy sparse matrices then?

Or only numpy arrays and scipy sparse matrices (since pandas DataFrame uses numpy arrays internally and assume the reader knows this fact)